docs/04-operation-guides/operations/obsv-03-configue-istio-access-logs.md

```diff
@@ -0,0 +1,53 @@
+---
+title: Configure Istio access logs
+---
+
+You can enable [Istio access logs](https://istio.io/latest/docs/tasks/observability/logs/access-log/) to provide fine-grained details about the workloads. This can help in indicating the golden signals, and troubleshooting anomalies.
```

Comment:
```suggestion
You can enable [Istio access logs](https://istio.io/latest/docs/tasks/observability/logs/access-log/) to provide fine-grained details about the workloads. This can help in indicating the four “golden signals” of monitoring (latency, traffic, errors, and saturation), and troubleshooting anomalies.
```

I just realized Istio themselves explain what the golden signals are, so we should do the same probably.